engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -36,9 +36,11 @@
 public abstract class Camera {
 
     /* CAMERA PARAMETERS */
-    protected final Vector3f position = new Vector3f(0, 0, 0);
+    protected final Vector3f position = new Vector3f();
     protected final Vector3f up = new Vector3f(0, 1, 0);
-    protected Vector3f viewingDirection = new Vector3f(1, 0, 0);
+    // FixMe this is clearly not the viewing direction, it's the rotation axis
+    // The usage of this value outside of this class should be thoroughly analyzed when fixing this.
```

Comment:
This has to be addressed. There need to be getters for both the actual viewing direction and the axis angle. Some clients are using the viewing axis, (most likely) thinking it is the viewing direction.

Reply:
Issue got resolved here: #3905